Packs/VirusTotal/Integrations/integration-VirusTotal_5.5.yml

```diff
@@ -282,8 +284,15 @@ script:
             } else {
                 dbotScore = 1;
             }
-            ec["DBotScore(val.Indicator && val.Indicator === obj.Indicator && val.Vendor === obj.Vendor && val.Type === obj.Type)"].push({Indicator: hash, Type: 'hash', Vendor: 'VirusTotal', Score: dbotScore});
-            ec["DBotScore(val.Indicator && val.Indicator === obj.Indicator && val.Vendor === obj.Vendor && val.Type === obj.Type)"].push({Indicator: hash, Type: 'file', Vendor: 'VirusTotal', Score: dbotScore});
+
+            ec[DBOTSCORE_KEY].push({Indicator: r[i].md5, Type: 'hash', Vendor: 'VirusTotal', Score: dbotScore});
+            ec[DBOTSCORE_KEY].push({Indicator: r[i].md5, Type: 'file', Vendor: 'VirusTotal', Score: dbotScore});
+
+            ec[DBOTSCORE_KEY].push({Indicator: r[i].sha1, Type: 'hash', Vendor: 'VirusTotal', Score: dbotScore});
+            ec[DBOTSCORE_KEY].push({Indicator: r[i].sha1, Type: 'file', Vendor: 'VirusTotal', Score: dbotScore});
+
+            ec[DBOTSCORE_KEY].push({Indicator: r[i].sha256, Type: 'hash', Vendor: 'VirusTotal', Score: dbotScore});
+            ec[DBOTSCORE_KEY].push({Indicator: r[i].sha256, Type: 'file', Vendor: 'VirusTotal', Score: dbotScore});
```

Comment:
aren't we pushing a list of indicators to the same entry?

Reply:
You're right, but it should be okay since it's only 3 indicators. It won't create a big entry, and they are aliases for the same indicator, see a screenshot from XSOAR:
![image](https://user-images.githubusercontent.com/20818773/97912396-6e9ada80-1d55-11eb-8854-bf73a67c94fc.png)